app/models/manageiq/providers/storage_manager/swift_manager/refresh_parser/cross_linkers.rb

```diff
@@ -5,6 +5,10 @@ def self.cross_link(ems, data)
       _log.warn "Manager does not have a parent."
       return
     end
+    unless data
+      _log.warn "Manager does not have any object storage."
```

Comment:
And just for perspective, under manageiq/app/models - there are 1373 calls to log methods using "(" and 642 not using them.  It is not consistent although there is a 2/1 ratio for.